tools/branch_comparisons.md

```diff
@@ -15,7 +15,7 @@ See the instructions on the
 
 * **C**: Citadel
 * **D**: Dome
-* **D**: Edifice
+* **E**: Edifice
 * **main**: main branch
```

Comment:
Does Fortress need to be added here and update the branch comparisons? https://github.com/ignitionrobotics/docs/pull/174#issuecomment-819813388

Reply:
I thought it needed to be updated, but then I realized I had already done it in https://github.com/ignitionrobotics/docs/pull/158/files#diff-376d38d0fcbaa70981a845c21214a3f839a976c24dd5e45175b38e04d7270b7a :sweat_smile: